gems/smithy-client/lib/smithy-client/signers/http_basic.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'base64'
+
```

Comment:
Do we need to require this here too? I noticed we already require this under `Smithy::Client` level

Reply:
I think it's best practice to require things where they are used even if they may have been required before. We know this file depends on base64 existing.